contrib/mesos/pkg/executor/executor.go

```diff
@@ -690,6 +644,10 @@ waitForRunningPod:
 
 				k.sendStatus(driver, statusUpdate)
 
+				if task.launchTimer.C != nil {
```

Comment:
agreed, i don't like this

On Wed, Dec 9, 2015 at 11:38 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/executor/executor.go
> <https://github.com/kubernetes/kubernetes/pull/17917#discussion_r47115429>
> :
>
> > @@ -690,6 +644,10 @@ waitForRunningPod:
> >
> >  				k.sendStatus(driver, statusUpdate)
> >
> > +				if task.launchTimer.C != nil {
>
> not nice, check launchTimer instead
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17917/files#r47115429>.
>